homeassistant/components/nut/__init__.py

```diff
@@ -79,6 +83,22 @@ async def async_update_data() -> dict[str, str]:
     await coordinator.async_config_entry_first_refresh()
     status = coordinator.data
 
+    all_device_supported_commands = data.list_commands() or {}
+    supported_commands: set[str] = set()
```

Comment:
I would probably write this as a comprehension since I don't think there is much value in the debug logging as long as it's covered by a test

Reply:
The intent was to help to troubleshoot in case that some expected commands were not shown in the UI as available options, but we can add it back later on if it ends up being an issue.